opentelemetry-aws/Cargo.toml

```diff
@@ -28,5 +28,5 @@ lazy_static = "1.4"
 [dev-dependencies]
 opentelemetry = { path = "../opentelemetry", features = ["trace", "testing"] }
 opentelemetry-http = { path = "../opentelemetry-http" }
-hyper = { version = "0.14", features = ["full"] }
-tokio = { version = "1.0", features = ["full"] }
+hyper = { version = "0.14" }
```

Comment:
Should we explicitly do `default-features = false` here, and for tokio?

Reply:
We could, but I'm not sure that it is very important to do so, as this is just for dev-dependencies, so generally doesn't pollute dependant crates.